lib/cryptoexchange/exchanges/tron_trade/market.rb

```diff
@@ -0,0 +1,15 @@
+module Cryptoexchange::Exchanges
+  module TronTrade
+    class Market < Cryptoexchange::Models::Market
+      NAME = 'tron_trade'
+      API_URL = 'https://trontrade.io/graphql'
+
+      def self.trade_page_url(args={})
+        client = Cryptoexchange::Client.new
+        pair = Cryptoexchange::Models::MarketPair.new(base: args[:base], target: args[:target], market: 'tron_trade')
+        ticker = client.ticker(pair)
+        "https://trontrade.io/exchange/#{ticker.payload["id"]}"
```

Comment:
lets drop support for this, we dont want to make an API call everytime this method is accessed.